crmd/lrm.c

```diff
@@ -2317,6 +2317,26 @@ do_lrm_event(long long action,
     CRM_CHECK(FALSE, return);
 }
 
+static char *
+remove_escaped_newlines(const char *string)
+{
+    char *pch = NULL;
+    char *ret = NULL;
+    static const char *escaped_newline = "\\n";
+
+    if (!string) {
+        return NULL;
+    }
+
+    ret = strdup(string);
+
+    while ((pch = strstr(ret, escaped_newline)) != NULL) {
+        memmove(pch, pch + strlen(escaped_newline), strlen(pch + strlen(escaped_newline)) + 1);
+    }
+
+    return ret;
+}
```

Comment:
Backslash-n isn't a valid XML sequence, so it would be better if we could avoid adding it to begin with. Are they in the metadata supplied by the agent, or is pacemaker adding them?

memmove() is inefficient -- if we have to keep this, let's just replace the characters with spaces.